apps/src/lab2/views/components/editor/editorConfig.ts

```diff
@@ -24,6 +31,19 @@ import {
   rectangularSelection,
 } from '@codemirror/view';
 
+// These are the almost same as the default keybindings for autocomplete,
```

Comment:
I'm sure you already thought of this, but is there any way to keep the defaultKeymap and only override the 'Tab' one, to avoid having to have all the other defaults in this autocompleteKeybindings list?

Reply:
Unfortunately no, what they export is read-only.